DEPS

```diff
@@ -21,6 +21,7 @@ deps = {
   "third_party/bitcoin-core/src": "https://github.com/bitcoin/bitcoin.git@95ea54ba089610019a74c1176a2c7c0dba144b1c",
   "third_party/argon2/src": "https://github.com/P-H-C/phc-winner-argon2.git@62358ba2123abd17fccf2a108a301d4b52c01a7c",
   "third_party/rapidjson/src": "https://github.com/Tencent/rapidjson.git@06d58b9e848c650114556a23294d0b6440078c61",
+  "components/playlist/browser/resources/media_detector": "https://github.com/brave/playlist-component@58639d1d08bf6aaf3c92124c86db57332016a4c4",
```

Comment:
maybe condition should be all platform except ios?

Reply:
Does iOS add the script some other way? (sorry, I'm pretty hazy on how gclient works)

Reply:
Yeah, excluding ios from this would be great. iOS has their own iOS script in their repo, so they are not counting on playlist-component repo at all. In the future I hope we can share the same scripts :)